lang/src/main/java/org/apache/shiro/lang/io/ClassResolvingObjectInputStream.java

```diff
@@ -27,7 +27,8 @@
 import java.io.ObjectStreamClass;
 
 /**
- * Enables correct ClassLoader lookup in various environments (e.g. JEE Servers, etc).
```

Comment:
This file must not be part of the commit. Please rebase and create the PR from a branch.